frontend/src/components/CriteriaBarChart.tsx

```diff
@@ -123,10 +129,15 @@ const CriteriaBarChart = ({ video }: Props) => {
         content={(props: TooltipProps<number, string>) => {
           const payload = props?.payload;
           if (payload && payload[0]) {
-            const { criteria, score } = payload[0].payload;
+            const { criteria, clippedScore, personalScore } =
+              payload[0].payload;
             return (
               <pre>
-                {getCriteriaLabel(criteria)}: {displayScore(score)}
+                {getCriteriaLabel(criteria)}: {displayScore(clippedScore)}
```

Comment:
I think we want to display the true score here, not the clippedScore. The score is clipped only for display... Now that I think of it, I'm not absolutely sure about what we exactly want. Maybe it would help if we had som indications that bars are clipped for very good scores?

Otherwise to be the most honest we could display something like:

"Displayed as 10.0; Real score is 17.4"

Reply:
I changed back to displaying the non clipped score.